File: Task/Conjugate-a-Latin-verb/F-Sharp/conjugate-a-latin-verb.fs
// Conjugate a Latin Verb. Nigel Galloway: September 17th., 2021
let myLatin (n:string)=printfn "Rogatus sum iungere verbum %s" n
                       match n.Length>3,n.[-3..]="are" with
                        (false,_)|(_,false)->printfn "    facis quod"
                       |_->["o";"as";"at";"amus";"atis";"ant"]|>List.iter(fun g->printfn "    %s%s" n.[0.. -4] g)
myLatin "amare"
myLatin "creo"

File: Task/Conjugate-a-Latin-verb/PascalABC.NET/conjugate-a-latin-verb.pas
// Conjugate a Latin verb. Nigel Galloway: September 30th., 2024
##
procedure myLatin(n:string);
begin
  println($'Tu quae conjugate verba {n}');
  if (n.Length<4) or (n[n.Length-2:]<>'are') then begin println('Te adiuvare non possum'); exit; end;
  |'o','as','at','amus','atis','ant'|.ForEach(g->println($'{n[:n.Length-2]}{g}'));
end;

myLatin('amare');
myLatin('creo');
